app/controllers/accounts_controller.rb

```diff
@@ -0,0 +1,33 @@
+class AccountsController < ApplicationController
+    def index
+        limit = params[:size] || 10
+        offset = params[:page] ? (limit * page) : (limit * 0)
+        @accounts = Account.limit(limit).offset(offset)
```

Comment:
You can use certain gems for pagination.
https://github.com/kaminari/kaminari
https://github.com/mislav/will_paginate
https://github.com/ddnexus/pagy
Not necessary though, but in a real app it would be better.